config/gradle/common.gradle

```diff
@@ -25,6 +25,13 @@ repositories {
     }
 }
 
+dependencies {
```

Comment:
@msteiger - curiously where'd this come from? It looks unusual. I guess it sets the version to use but the format looks different?